public/javascripts/SVLabel/src/SVLabel/Tracker.js

```diff
@@ -63,6 +63,10 @@ function Tracker () {
     };
 
     this._isDeleteLabelAction = function (action) {
+        return action.indexOf("RemoveLabel") >= 0;
+    };
+
+    this._isClickLbelDeleteAction = function (action) {
```

Comment:
Oops, a typo in the name of the function! `_isClickLbelDeleteAction` -> `_isClickLabelDeleteAction`